src/GZCTF/Program.cs

```diff
@@ -400,17 +402,15 @@ public static IActionResult InvalidModelStateHandler(ActionContext context)
                 return new JsonResult(
                     new RequestResponse(errors is [_, ..]
                         ? errors
-                        : StaticLocalizer[nameof(Resources.Program.Model_ValidationFailed)]))
-                { StatusCode = 400 };
+                        : StaticLocalizer[nameof(Resources.Program.Model_ValidationFailed)])) { StatusCode = 400 };
 
             errors = (from val in context.ModelState.Values
-                      where val.Errors.Count > 0
-                      select val.Errors.FirstOrDefault()?.ErrorMessage).FirstOrDefault();
+                where val.Errors.Count > 0
+                select val.Errors.FirstOrDefault()?.ErrorMessage).FirstOrDefault();
 
             return new JsonResult(new RequestResponse(errors is [_, ..]
                 ? errors
-                : StaticLocalizer[nameof(Resources.Program.Model_ValidationFailed)]))
-            { StatusCode = 400 };
+                : StaticLocalizer[nameof(Resources.Program.Model_ValidationFailed)])) { StatusCode = 400 };
```

Comment:
It seems like a mistake existed even before your change. This should not use `StaticLocalizer`. Can you change it to use the one from DI instead?
To get the localizer from DI, you can call `context.RequestServices.GetRequiredService<IStringLocalizer<Program>>()`.